cotk/dataloader/multi_turn_dialog.py

```diff
@@ -421,7 +428,7 @@ def tokenize(self, sentence):
 		Returns:
 			sent (list): list of token(str)
 		'''
-		return WordPunctTokenizer().tokenize(sentence.replace('__eou__', ''))
+		return WordPunctTokenizer().tokenize(sentence)
```

Comment:
Call the tokenizer from LanguageProcessingBase (so we can adapt to different files.)

Reply:
Refer to SingleTurnDialog::tokenize